src/entities/Category/model.ts

```diff
@@ -19,13 +24,18 @@ export default class CategoryModel extends Model<CategoryAttributes> {
     )
   }
 
-  static async findActiveCategoriesWithPlaces(): Promise<
-    CategoryWithPlaceCount[]
-  > {
+  static async findActiveCategoriesWithPlaces(
+    target: "all" | "worlds" | "places" = "all"
```

Comment:
With the enum this would be

```suggestion
    target: CategoryCountTargetOptions = CategoryCountTargetOptions.ALL
```